### DISCUSSION TITLE
Не работает кеширование

### DISCUSSION THREAD

**Discussion by SemaEvdo** [AdguardTeam/AdGuardHome#6155] [Q&A]
Добрый день. Спасибо за хороший продукт!

Обнаружил интересную вешь: при установке своего (кастомного апстрима) клиенту, перестаёт работать кеширование его запросов.

Это баг или фича? :)

![1](https://github.com/AdguardTeam/AdGuardHome/assets/111609344/dd69602a-e8d1-4514-b6e0-1165077f20b7)
![2](https://github.com/AdguardTeam/AdGuardHome/assets/111609344/9a3e98fa-1077-46b8-b6f4-87fee0b90ef9)

Заранее благодарю за ответ.

**Comment by ainar-g** [comment]
Добрый день.  Это известная проблема одной из библиотек.  См. https://github.com/AdguardTeam/dnsproxy/issues/169#issuecomment-930944577.
